packages/affine/block-paragraph/src/paragraph-drag-extension.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  ParagraphBlockModel,
+  ParagraphBlockSchema,
+} from '@blocksuite/affine-model';
+import { DragHandleConfigExtension } from '@blocksuite/affine-shared/services';
+import {
+  calculateCollapsedSiblings,
+  captureEventTarget,
+  matchFlavours,
+} from '@blocksuite/affine-shared/utils';
+
+export const ParagraphDragHandleOption = DragHandleConfigExtension({
```

Comment:
This `DragHandleConfigExtension` is not used in new drag and drop.

Reply:
What should I use now?

Reply:
emmm, the new drag and drop havn't support extending now and you need to write the logic in [drag-event-watcher](https://github.com/toeverything/blocksuite/blob/master/packages/blocks/src/root-block/widgets/drag-handle/watchers/drag-event-watcher.ts)